pyspecProcScripts/phasing.py

```diff
@@ -242,16 +238,20 @@ def hermitian_function_test(
     orig_dt = s.get_ft_prop(direct, "dt")
     s.ft(direct)
     s = s[direct:frq_range]
-    s.ift(direct, pad=2048 * 8)
+    s.ift(direct, pad=1024 * 16)
+    new_dt = s.get_ft_prop(direct, "dt")
+    non_aliased_range = r_[3,-3]*int(orig_dt/new_dt)
+    ini_delay = non_aliased_range[0]*new_dt
+    s = s[direct,non_aliased_range[0]:non_aliased_range[1]]
```

Comment:
as noted earlier, this convention replaced both the `ini_delay` kwarg and the `selection_range` kwarg.